src/main.rs

```diff
@@ -1,3 +1,22 @@
-fn main() {
-    println!("Hello, Gorillas!");
+extern crate ggez;
+use ggez::*;
```

Comment:
Similar notes as above.

I also noticed, even in the old (2015) edition, the `extern crate` was needed only in the top-level module, not the submodules.

Reply:
Thank you, fixed in 5f8ce91bdb059c9cf4bd0b28de3c4febcc81bb47